.gitignore

```diff
@@ -1,2 +1,3 @@
 .venv
-/images
\ No newline at end of file
+/images
+.env
```

Comment:
could you also add `*.csv ` to the gitignore so that we aren't pushing the data as it has the botanist details on there.